src/transformations/utils/build-super-calls-for-chain.ts

```diff
@@ -88,15 +88,13 @@ export function buildSuperCallsForChain(
     if (parentNode !== contractNode) {
       // step 1 check if initializable
       const args = ctorCalls[parentNode.id]?.call?.arguments;
+      // To be initializable means that said parent has all the variables needed for the constuctor
       const initializable = args !== undefined || isImplicitlyConstructed(parentNode);
       if (!initializable) {
-        // step 2 get the parent parents
-        const parents = parentNode.linearizedBaseContracts;
-
-        // step 3 add them to the list
-        parents.map(id => {
-          notInitializable.add(id);
-        });
+        // step 2 get the parent parents and add them to the list
```

Comment:
To make comments valuable, we have to try and explain the why of things, which is sometimes very implicit in the code:

```suggestion
        // If a parent is not initializable, we assume its parents aren't initializable either,
        // because we may not have their constructor arguments.
        // The user will invoke them anyway in the chained initializer of this parent, which
        // will have to be manually called.
```